.github/workflows/tests.yml

```diff
@@ -85,7 +85,7 @@ jobs:
         run: python -m pip install -i https://test.pypi.org/simple/ PennyLane-Lightning --pre --upgrade
 
       - name: Run tests
-        run: python -m pytest tests --cov=pennylane $COVERAGE_FLAGS -m "$SUITE" -n auto
+        run: python -m pytest tests --cov=pennylane $COVERAGE_FLAGS -m "$SUITE and not qcut" -n auto

```

Comment:
> I also now realize that certain qcut tests are being run in the interface-specific CI runners in addition to the qcut specific CI runner. I modified the github actions so that interface suites don't run qcut tests again.

So this is for turning off the interface-suite tests? :thinking:

Reply:
Yep. This way qcut tests will only run once in the qcut runner.